doc/specs/vulkan/chapters/resources.txt

```diff
@@ -3958,6 +3958,12 @@ ifdef::VK_KHR_swapchain[]
     slink:VkBindImageMemorySwapchainInfoKHR, pname:memory must: be a valid
     dname:VkDeviceMemory handle
 endif::VK_KHR_swapchain[]
+ifndef::VK_KHR_swapchain[]
+  * pname:memory must: be a valid dname:VkDeviceMemory handle
+endif::VK_KHR_swapchain[]
+endif::VK_VERSION_1_1,VK_KHR_device_group[]
+ifndef::VK_VERSION_1_1,VK_KHR_device_group[]
```

Comment:
I believe the fallthrough VU at line 3966 replicates VUID-VkBindImageMemoryInfo-memory-01625 above.

Reply:
Yes, the VU aplies to two conditions:
- `VK_KHR_device_group && !VK_KHR_swapchain`
- `!VK_KHR_device_group`

Such condition cannot be implemented using current (documented) `ifdef` as a single VU.

Reply:
Alright, removed all duplicates.